tests/lib/gladecheck.py

```diff
@@ -78,107 +56,30 @@ def checkGlade(self, glade_tree):
         """
         pass
 
-    def test_glade_file(self):
+    def check_glade_files(self):
```

Comment:
It should be `_check_glade_files`. Also, I would consider to remove `checkGlade` from this class and explicitly specify it as an argument of this method to make the tests easier to understand. That will allow to have run more checks in one test case.